css/styles.css

```diff
@@ -0,0 +1,85 @@
+.notes-input textarea {
+  height: 150px;
+  resize: none;
+}
+
+.column-half {
+  padding-bottom: 15px;
+}
+
+.title {
+  padding-bottom: 15px;
+}
+
+.notes-label,
+.title-label,
+.photo-url-label {
+  margin-bottom: 5px;
+}
+
+#notes,
+#title,
+#photo-url {
+  width: 100%;
+  border: 1.5px solid rgb(182, 182, 182);
+  border-radius: 3px;
+}
+
+#notes:focus,
+#title:focus,
+#photo-url:focus {
+  outline: none;
+  border: 1.5px solid #562b81;
+}
+
+#notes:hover,
+#title:hover,
+#photo-url:hover {
+  outline: none;
+  border: 1.5px solid #562b81;
+}
```

Comment:
Please use classes (instead of ids) for CSS targets.

Reply:
got it 👍